gradle.properties

```diff
@@ -17,4 +17,6 @@ org.gradle.jvmargs=-Xmx2048m -Dfile.encoding=UTF-8
 android.useAndroidX=true
 
 # Kotlin code style for this project: "official" or "obsolete":
-kotlin.code.style=official
\ No newline at end of file
+kotlin.code.style=official
+
+detektVersion=1.18.0-RC2
```

Comment:
The detekt version needed to be added in gradle.properties due to this gradle plugin limitation: https://github.com/gradle/gradle/issues/1697#issuecomment-506910915